src/components/executive-education-2u/styles/_ExecutiveEducation2UErrorPage.scss

```diff
@@ -0,0 +1,61 @@
+$heading-red: #D23228;
+$heading-black: #002121;
+$error-chip-text-color: #002121;
+$error-chip-background-color: #FFFFFF;
+$error-chip-border-color: #F2F0EF;
+
+.executive-education-2u-error-illustration {
+    @extend .mb-4;
```

Comment:
Would not recommend using `@extend` as it will include an `!important`, which makes theming difficult. Use `margin-bottom: map-get($spacers, 4);` instead. Or, apply the `.mb-4` utility class (preferred).